src/fileWriter.ts

```diff
@@ -25,17 +25,31 @@ const write = (path: string, data: OutputData) => rw.writeFileSync(
   'utf8',
 )
 
+const cardanoEraToSignedType = {
```

Comment:
We can type this like this `{[key in CardanoEra]: string}`.

Reply:
Fixed